util_problematic_word_extraction.py

```diff
@@ -0,0 +1,117 @@
+# Extract potentially problematic words for TTS from a CSV using a spell checker
+# Divide the words up into separate files by punctuation
+
+import pandas as pd
+# may need to do a 'pip install pyspellchecker'
+from spellchecker import SpellChecker 
+# https://pyspellchecker.readthedocs.io/en/latest/
+import sys
+
+# python problematic_word_extraction.py messages.csv
+
+spell = SpellChecker()
```

Comment:
I would like to expose an additional parameter so we can support non-English languages:
https://pyspellchecker.readthedocs.io/en/latest/#non-english-dictionaries

Reply:
added